src/jvm/main/org/jetbrains/kotlinx/lincheck/IdeaPlugin.kt

```diff
@@ -36,14 +39,16 @@ const val MINIMAL_PLUGIN_VERSION = "0.2"
  * @param version current Lincheck version.
  * @param minimalPluginVersion minimal compatible plugin version.
  * @param exceptions representation of the exceptions with their stacktrace occurred during the execution.
+ * @param isGeneralPurposeModelChecking indicates if lincheck is running in the GPMC mode
  */
 @Suppress("UNUSED_PARAMETER")
 fun testFailed(
     failureType: String,
     trace: Array<String>,
     version: String?,
     minimalPluginVersion: String,
-    exceptions: Array<String>
+    exceptions: Array<String>,
+    isGeneralPurposeModelChecking: Boolean,
```

Comment:
NOTE: we will have to replace it with `enum` when adding the Trace Debugger mode